istioctl/pkg/waypoint/waypoint.go

```diff
@@ -158,12 +162,38 @@ func Cmd(ctx cli.Context) *cobra.Command {
   istioctl x waypoint delete --service-account something --namespace default
 
   # Delete a waypoint by name, which can obtain from istioctl x waypoint list 
-  istioctl x waypoint delete waypoint-name --namespace default`,
+  istioctl x waypoint delete waypoint-name --namespace default
+
+  # Delete several waypoints by name
+  istioctl x waypoint delete waypoint-name1 waypoint-name2 --namespace default
+
+  # Delete all waypoints in a specific namespace
+  istioctl x waypoint delete --all --namespace default`,
```

Comment:
what's the behavior if user does `istioctl x waypoint delete --all -A`? I imagine that is not a scenario we do not want to support?

Reply:
Yes, it's not supported in `delete` command, it's only supported in `list` command.

Reply:
```
istioctl x waypoint delete  -n bookinfo --all -A
Error: unknown shorthand flag: 'A' in -A
```

Reply:
sounds good